style.css

```diff
@@ -452,3 +417,355 @@ button {
   background-color: #fff;
   color: var(--neutral5);
 }
+
+/* *************************** Desktop Version ***************************************** */
+@media (min-width: 992px) and (max-width: 1100px) {
```

Comment:
- [ ] I think this media query is meant to be `@media screen and (min-width: 1080px) {}` because with yours this desktop design only happens between 992px to 1100px and then falls apart which is not right. I believe a responsive design should look good on all screen sizes.
- [ ] I also request that you fix the mobile view for 👇🏻 

```
@media screen and (min-width: 990px) {

}
```

Reply:
Hi @Cathella 
Thanks for your feedback. There are some points that are not so clear:
- Desktop version should starts at 1080px `@media screen and (min-width: 1080px)`
- Movile version should starts at `@media screen and (min-width: 990px)`, shouldn't be `@media screen and (max-width: 990px)`
- What should be the media queries for mobile version?
 I really appreciate your time and effort. Thanks